common/constants/integrations.ts

```diff
@@ -5,7 +5,3 @@
 
 export const OPENSEARCH_DOCUMENTATION_URL = 'https://opensearch.org/docs/latest/integrations/index';
 export const ASSET_FILTER_OPTIONS = ['index-pattern', 'search', 'visualization', 'dashboard'];
```

Comment:
if it is easy can we also include the asset filter not being hard coded?

Reply:
Should be identical to how we removed the template list, though since we're repeating the deduplicate+sort logic a bunch we should try to centralize it to a general method somewhere. Can look into it later

Reply:
Update: Since due dates are shifted let's just merge the current version and deal with this in a separate PR @derek-ho